source/versions/versioning.rst

```diff
@@ -4,115 +4,111 @@ Versioning
+a number, this build is a **recommended version**. Any plugin developed for the SpongeAPI version in the string, or is  
+compatible with that version, is guaranteed to work.
 
-.. tip::
+.. note::
+
+    The *Patch* number is always reset to zero when the *Minor* number changes.
+
+Examples of implementation version strings are ``spongevanilla-1.12.2-7.1.5`` and ``spongeforge-1.12.2-2768-7.1.5``. 
+
+.. warning::
+
+    Plugin developers may choose to develop their plugin for a particular implementation. The plugin's Ore page should 
+    note such decision with a tag by the version. Sponge makes no guarantee of a plugin's compatibility when marked as 
+    such.
+
+Recommended Version
+===================
 
-    ``spongevanilla-1.12.2-7.2.0`` is a released version of the API and a released version of the implementation. It is 
-    also a recommended version because the *RC* label is not used.
+Recommended Versions are also known as Recommended Builds or Releases. They are versions of reasonable quality of which 
+the implementations can make full use of the functionality available in SpongeAPI, and plugins compatible with the API 
+are guaranteed to work as designed.
+
+.. warning::
+
+    The following information is for Sponge Developers. Plugin Developers and Server Administrators should only use 
+    **Recommended Versions** for development and server installations. Doing otherwise is at your own risk and **not** 
+    encouraged.
+
+SNAPSHOT
+========
+
+A release which has the ``-SNAPSHOT`` label represents the next API version in development. For example, 
+``7.2.0-SNAPSHOT`` means ``7.2.0`` is in development. Another example is ``8.0.0-SNAPSHOT``, which means the next 
+*major* release (``8.0.0``) is in development. New features added in snapshots may break at any time. 
+
+.. note::
+
+    If the minor version is zero (e.g., ``8.0-SNAPSHOT``), **anything** may break before it is released and plugins 
+    cannot expect stability. If the Minor version is greater than zero (e.g., ``7.2-SNAPSHOT``), anything up to the 
+    previous minor version is still guaranteed to work (in this case, ``7.1``). However, anything added since it **may 
+    break** and instability is likely.
+
+    When the minor and patch numbers are zero, and the ``-SNAPSHOT`` label is attached, versions are ``bleeding`` 
+    builds for testing the next version of SpongeAPI only and may break at any time.
+
+    The ``-SNAPSHOT`` label on a SpongeCommon file means the implementation is nearly complete but is still not a 
+    stable release. More importantly, Gradle determines whether or not to append the *-RC{build number}* to the version 
+    string by the presence of the label.
+
+Release Candidate
+=================
+
+Release Candidates (**RC**) are builds in which bug fixes are released for implementations while they are still in 
+development. They are nearly ready for a full release, but require more bug fixes and testing.
```

Comment:
as we run rolling releases, every commit is a RC!
they can range from not even launching to just as stable as a rb

Reply:
I don't understand what you're saying. Should I remove this section, add a statement about the rolling releases, or am I missing the release candidate idea completely?

Reply:
Yes, anything that isn't an RB is an RC, it isn't just stuff that isn't ready, it's everything that gets pushed to Github doesn't matter what state it is in. Any RC can become an RB: e.g. these 2 builds are identical from the codebase https://i.imgur.com/m5iLnvG.png